third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -1696,6 +1696,14 @@ struct AsyncTMAGatherOpConversion
     : public ConvertOpToLLVMPattern<triton::nvidia_gpu::AsyncTMAGatherOp> {
   using ConvertOpToLLVMPattern::ConvertOpToLLVMPattern;
 
+  AsyncTMAGatherOpConversion(LLVMTypeConverter &converter,
+                             PatternBenefit benefit, int computeCapability)
+      : ConvertOpToLLVMPattern<triton::nvidia_gpu::AsyncTMAGatherOp>(converter,
+                                                                     benefit),
+        computeCapability(computeCapability) {}
+
+  int computeCapability;
```

Comment:
we can revert that too?

Reply:
Thanks for catching what I missed! fixed!